onboarding-manager/internal/tinkerbell/templates/microvisor.yaml

```diff
@@ -30,6 +30,9 @@ tasks:
           IMG_URL: {{ .DeviceInfoOSImageURL }}
           SHA256: {{ .DeviceInfoOsImageSHA256 }}
           COMPRESSED: true
+          HTTP_PROXY: {{ .EnvENProxyHTTP }}
+          HTTPS_PROXY: {{ .EnvENProxyHTTPS }}
+          NO_PROXY: {{ .EnvENProxyNoProxy }}
```

Comment:
what if the proxy are coming from cloud init?

Reply:
This is only for image download similar to what's done for the Ubuntu image. I am missing something in your question probably. Do you intend to say that different EN could be in different proxy env (configured via custom cloud-init)?

Reply:
with the custom cloud-init we have the ability to configure specific EN proxy settings. `Env` Proxy are defined at the cluster installation - which ones are we supposed to use ? Do we expect any merging strategy ? This applies also for the final OS configuration

Reply:
env proxies here are used only in the MicroOS stage. The custom cloud-init works for the target OS only. No conflict here. Users are not allowed to pass custom cloud-init to the MicroOS.